### PR TITLE
Add portfolio accessibility

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Portfolio Microverse Project  🚀 

 > This PR adds a pair programming exercise to evaluate the portfolio project against accessibility best practices and implement any necessary changes to ensure that the website is accessible.

## Web accessibility evaluation
- [ ] Page titles - Short description of the title
https://github.com/bimbolabuari/mv-portfolio/blob/4e1754c296c46c8fc8b18e419e5d1ddc7d8ad17a/index.html#L6
- [ ] Image text alternatives - No issues found
- [ ] Text headings - No issues found
- [ ] Color contrast - Low contrast between text and background of work-cards.
https://github.com/bimbolabuari/mv-portfolio/blob/4e1754c296c46c8fc8b18e419e5d1ddc7d8ad17a/styles.css#L361
- [ ] Resize - No issues found
- [ ] Interaction - Key tab doesn't work in the navbar.
https://github.com/bimbolabuari/mv-portfolio/blob/4e1754c296c46c8fc8b18e419e5d1ddc7d8ad17a/index.html#L23
- [ ] Moving content - No issues found
- [ ] Multimedia - No issues found
- [ ] The basic structure of the page - No issues found

## Live Preview  🎉 

[Live Preview Link](https://deploy-preview-6--bimbola-mv-portfolio.netlify.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
+    <title>Portfolio-Software developer Portfolio</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
       type="text/css"
     />
-    <link href="styles.css" rel="stylesheet" />
+    <!-- <link href="styles.css" rel="stylesheet" /> -->
```
Comment (vichuge) [inline index.html]: - [x] Please uncoment the style 🐶

File: index.html [hunk 2]
```diff
@@ -3,27 +3,27 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>Portfolio</title>
+    <title>Portfolio-Software developer Portfolio</title>
```
Comment (vichuge) [inline index.html]: - [x] I recommend you to use adequately and briefly describes the content of the page. For example:
- Mark's portfolio
- Mark full-stack developer
- 🐱

File: styles.css [hunk 3]
```diff
@@ -361,7 +358,7 @@
 }
 
 .project-container {
-  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("././images/project-setup3.jpg");
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.9) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("././images/project-setup3.jpg");
```
Comment (vichuge) [inline styles.css:365]: - [ ] I understand to use an image as a background could be helpful, at the same time, use it on `` inside HTML code is part of this project to understand how images work with this important tag. I invite you to, some of the images you use on the project, include them on HTML code with the `` tag. I'm pretty sure you'll get another way to work with images on this way. 🐶

File: index.html [hunk 4]
```diff
@@ -3,27 +3,27 @@
...
       </nav>
       <ul class="nav-list block">
-        <li class="ul-list cursor"><a>Portfolio</a></li>
-        <li class="ul-list cursor"><a>About</a></li>
-        <li class="ul-list cursor"><a>Contact</a></li>
-        <li class="ul-list cursor"><a><i class="fa fa-envelope white header-icon"></i></a></li>
+        <li class="ul-list cursor"><a href="#">Portfolio</a></li>
+        <li class="ul-list cursor"><a href="#">About</a></li>
+        <li class="ul-list cursor"><a href="#">Contact</a></li>
+        <li class="ul-list cursor"><a href="#"><i class="fa fa-envelope white header-icon"></i></a></li>
```
Comment (vichuge) [inline index.html]: - [x] The color for this is hard to get. I invite you to change it 🐦 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @bimbolabuari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- Good Linters ✔️ 
- I like your project once the style is activated ✔️ 

## Aspects to improve ♻️

- [x] The page should have a minimum contrast ratio of at least 4.5:1 for normal-size text.
- You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.
- Because the styles aren't activated, I don't know if this works, please, check it once the style is activated.
- [x] It's important to add `<label>` associated with every `<input>` on `<form>`, I kindly invite you to add it.
- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bimbolabuari** [comment]
@vichuge, Thanks for the review, I have made the required changes 🙂

File: index.html (hunk 4 above)
Comment (Davidosky007) [reply index.html]: done 👍

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html]: done 👍

File: index.html (hunk 2 above)
Comment (Davidosky007) [reply index.html]: done 👍

**Comment by Davidosky007** [comment]
Hi @bimbolabuari, 

# Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 
CR1 GIF
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bimbolabuari** [comment]
@Davidosky007, Thanks for approving my project 🎉

**Pull request merged**